data/mods/Magiclysm/eoc_spell_learn_boost.json

```
@@ -0,0 +1,140 @@
+[
+  {
+    "type": "jmath_function",
+    "id": "mc_spell_exp_diff",
+    "num_args": 1,
+    "//": "accept the spell level, return a difference in experience between spell's current level and the next level",
+    "return": "spell_exp(_0 + 1) - spell_exp(_0)"
+  },
+  {
+    "type": "jmath_function",
+    "id": "mc_spell_train_focus_factor",
+    "num_args": 2,
+    "//args": "_0=focus, _1=spell_level",
+    "//": "At level 15 focus factor is enabled, 50 focus = twice more spellcasts, 100 focus = same, 200 focus = twice less spellcasts",
+    "return": "_1 > 15 ? (max(20, _0) - 50) / 100 + 0.5 : 1"
+  },
+  {
+    "type": "jmath_function",
+    "id": "mc_spell_cost_train_factor",
+    "num_args": 3,
+    "//args": "_0=cost,_1=min_spellcasts,_2=max_spellcasts",
+    "//": "accept the spell _cost, return how much spellcasts required to level the spell; formula assumes 10 spellcasts at 100% manapool cost, and 2 spellcasts at 1% manapool cost",
```

```suggestion
    "//": "accept the spell _cost, return how many spellcasts are required to level the spell; formula assumes 10 spellcasts at 100% manapool cost, and 2 spellcasts at 1% manapool cost",
```
Am I reading this correctly that spells that cost 100% of the players mana total will level in 10 castings and spells that use 1% of the players total mana will level every 2 times you cast it?

---

No, opposite. You have 1500 mana pool, and spell costs 1500 mana - you need 2 spellcasts to level it up. If spell costs 15 mana - you need 10 spellcasts.

---

I didn't notice that I had mixed up the values in places.